XenAdmin/Core/VMGroup.cs

```diff
@@ -158,7 +158,13 @@ internal static string UpsellLearnMoreUrl
             get { return typeof(T) == typeof(VMSS) ? InvisibleMessages.UPSELL_LEARNMOREURL_VMSS : InvisibleMessages.UPSELL_LEARNMOREURL_VM_APPLIANCES; }
         }
 
-        internal static AsyncAction AssignVMsToGroupAction(T group, List<XenRef<VM>> vms, bool suppressHistory)
+        internal static AsyncAction AssignVMsToGroupAction(T group, List<XenRef<VM>> vmRefs, bool suppressHistory)

```

Comment:
I will revert these changes, as it will be more convienent to have a list of XenRefs in the action in order to calculate the set difference between two lists more efficiently.